commands/api.go

```diff
@@ -728,6 +745,34 @@ func getLargestApiNameSize(retApiArray *whisk.RetApiArray, apiPath string, apiVe
 	return maxNameSize
 }
 
+func validatePathParameters(relativePath string) error {
```

Comment:
Are path parameters JSON? Can we just attempt a JSON parse instead?

Reply:
Nope, this is like above, where the string is `/my/{param}/path`.